scripts/combine_tables.R

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env Rscript
 
-# BackBLAST_combine_BLAST_tables.R
+# combine_tables.R
```

Comment:
All scripts have been renamed so that their names are simpler. No more `BackBLAST` prefix, for example.